modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -36,57 +36,44 @@ Built-in dashboards include various pages to view different types of charts:
 [%header,cols="1,4"]
 |===
 | Built-In Dashboards | Charts
-| *Overview* a|
+| <<overview-charts>> a|
 
 * Inbound - Total Requests by Response Type
 * Inbound - Average Response Time
-* Inbound - Mule messages
+* Mule Messages
 * Outbound - Total Requests by Response Type
 * Outbound - Average Response Time
 * JVM - CPU utilization
 * JVM - Heap memory utilization
 * JVM - Thread count
 
-//* Response Time by App: Number of milliseconds (ms) required for each response.
-//* Messages Processed: Number of messages at a given time.
-//* Response Time by Inbound Endpoint: Number of milliseconds (ms) for a response from inbound endpoints to your Mule app.
-//* Response Time by Outbound Endpoint: Number of milliseconds (ms) for a response from outbound endpoints of your Mule app.
-//* CPU: Percentage of CPU used over time. The percentages are calculated per worker and as the total for all workers.
-//* Memory: Number of mebibytes (MiB) used over time.
-| *Inbound* a|
+| <<inbound-charts>> a|
 
 * Total Inbound Requests by Response Type
 * Average Response Time
-* Average Response Time by Endpoint
-** Average: Green if response time is less than 300ms; yellow if response time is greater than 300ms and less than 600ms; red if is greater than 600ms
-** Max: Green if response time is less than 300ms; yellow if response time is greater than 300ms and less than 500ms; red if is greater than 500ms
-** Min: Green if response time is less than 300ms; yellow if response time is greater than 300ms and less than 600ms; red if is greater than 600ms
+* Response Time by Endpoint
 * Total Requests by Endpoint
-* Response Time graphs (99, 90, 75, and 50 percentile inbound): Measured in milliseconds (ms)
+* Response Time graphs (99, 90, 75, and 50 percentile)
 * Maximum Response Time
 * Total Failed Inbound Requests
```

Comment:
@myuin-ms UX title is "Failure Total Request Count". Should probably change the UX title to match this doc title.